services/apigateway/Apigateway.py

```diff
@@ -0,0 +1,76 @@
+import boto3
+import botocore
+import requests
+
+from utils.Config import Config
+from services.Service import Service
+from services.apigateway.drivers.ApiGatewayCommon import ApiGatewayCommon
+from services.apigateway.drivers.ApiGatewayRest import ApiGatewayRest
+
+class Apigateway(Service):
+   
+   
+    def __init__(self, region):
+        super().__init__(region)
+        ssBoto = self.ssBoto
+
+        self.apis = []
+        self.apisv2 = []
+        
+        self.apiClient = ssBoto.client('apigateway', config=self.bConfig)
+        self.apiv2Client = ssBoto.client('apigatewayv2', config=self.bConfig)
+        
+        return
+    
+    def getRestApis(self):
+        apis = []   
+
+        try:
+            apis = self.apiClient.get_rest_apis()
+            self.apis = apis.get('items')
+            while apis.get('position') is not None:
+                apis = self.apiClient.get_rest_apis(position=apis.get('position'))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `get_rest_apis` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `IsTruncated`, `NextToken`, `Position`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)